homework_03/05_Viktoriya_Vasileva/test_Circles.py

```diff
@@ -0,0 +1,74 @@
+from Circles import Point, Circle, RelativePosition
+
+
+def test_outside_circles():
+    circle_a = Circle(Point(1.0, 1.0), 1.0)
+    circle_b = Circle(Point(10.0, 10.0), 2.0)
+    assert circle_a.find_relative_position(circle_b) == RelativePosition.NO_COMMON_POINTS
+    
+    circle_a = Circle(Point(1.0, 1.0), 1.0)
+    circle_b = Circle(Point(10.0, 1.0), 7.0)
+    assert circle_a.find_relative_position(circle_b) == RelativePosition.NO_COMMON_POINTS
+
+    circle_a = Circle(Point(1.0, 1.0), 1.0)
+    circle_b = Circle(Point(10.0, 1.0), 7.99999)
+    assert circle_a.find_relative_position(circle_b) == RelativePosition.NO_COMMON_POINTS
+
+
+
+def test_touching_circles():
+    circle_a = Circle(Point(1.0, 1.0), 1.0)
+    circle_b = Circle(Point(1.0, 3.0), 1.0)
+    assert circle_a.find_relative_position(circle_b) == RelativePosition.TOUCHING
+    
+    circle_a = Circle(Point(2.3, 1.0), 1.0)
+    circle_b = Circle(Point(2.3, 9.0), 7.0)
+    assert circle_a.find_relative_position(circle_b) == RelativePosition.TOUCHING
+
+    circle_a = Circle(Point(1.0, 1.0), 1.0)
+    circle_b = Circle(Point(10.0, 1.0), 7.999999) #rounding
+    assert circle_a.find_relative_position(circle_b) == RelativePosition.TOUCHING
```

Comment:
This assert is the right one, I think that you just have not seen that you test two circles for two different values. Lines 28-30 are the same as 13-15. You can remove the useless situation from lines 13-15 when you merge your request.